integration/src/integration/java/pl/allegro/tech/hermes/integration/test/HttpResponseAssertion.java

```diff
@@ -20,6 +22,11 @@ public HttpResponseAssertion hasStatus(Response.Status status) {
         return this;
     }
 
+    public HttpResponseAssertion hasOneOfStatus(List<Response.Status> statuses) {
```

Comment:
How about adding here `hasSuccesStatus()` and in it just check whether the response is from 2xx family? Thanks to this you don't have to pass arguments like ACCEPTED, CREATED during the call, which simplifies a little bit reading and writing tests